src/components/LayoutTree.tsx

```diff
@@ -358,13 +358,15 @@ const LayoutTree = () => {
   const treeThoughts = useSelector(linearizeTree, _.isEqual)
   const fontSize = useSelector(state => state.fontSize)
   const dragInProgress = useSelector(state => state.dragInProgress)
+  const ref = useRef<HTMLDivElement | null>(null)
   const indentDepth = useSelector(state =>
     state.cursor && state.cursor.length > 2
       ? // when the cursor is on a leaf, the indention level should not change
         state.cursor.length - (hasChildren(state, head(state.cursor)) ? 2 : 3)
       : 0,
   )
 
+  const bulletWidth = ref.current?.querySelector('.bullet')?.getBoundingClientRect().width
```

Comment:
Correct me if I am not understanding it right, you are saying I should move the `getBoundingClientRect().width` inside a `useLayoutEffect` and only measure it when `dragInProgress` is true, right?

Reply:
Yes, that's correct.

I would recommend storing it in a ref rather than state, otherwise it will force a re-render.

Reply:
Alright, one issue I think could arise when only getting it on `dragInProgress` is if we have `simulateDrop` set to true, on first render the drop target for `b` and `d` would stretch all the way to col2 thoughts because `bulletWidth` would be 0 at that point. We need to set a default value in this case for it. I believe on font size 18 its bullet width is 28.09px.

Reply:
Right, I guess something like this would do fine then.

```
  useLayoutEffect(() => {
    bulletWidth.current =
      dragInProgress || testFlags.simulateDrop ? bullet?.getBoundingClientRect().width : bulletWidth.current
  }, [dragInProgress, bullet])
```

Reply:
Actually, I guess we do want it to re-render with the bulletWidth, so I was wrong about using a ref. Local state should work. Sorry for the confusion!

I'll leave it up to you to implement and confirm, then Trevin and I will review.